src/app/3d/qgs3dmapcanvaswidget.h

```diff
@@ -117,6 +118,7 @@ class APP_EXPORT Qgs3DMapCanvasWidget : public QWidget
     QToolButton *mBtnOptions = nullptr;
     QgsDockableWidgetHelper *mDockableWidgetHelper = nullptr;
     QgsRubberBand *mViewFrustumHighlight = nullptr;
+    QgsRubberBand *mViewExtentHighlight = nullptr;
```

Comment:
```suggestion
    QObjectUniquePtr< QgsRubberBand > mViewExtentHighlight;
```

likely is safer

Reply:
Done. I have also added a commit to change it for `mViewFrustumHighlight`.